src/main/java/org/opentripplanner/apis/gtfs/generated/graphql-codegen.yml

```diff
@@ -45,8 +45,8 @@ config:
     RentalVehicle: org.opentripplanner.service.vehiclerental.model.VehicleRentalVehicle#VehicleRentalVehicle
     VehicleRentalUris: org.opentripplanner.service.vehiclerental.model.VehicleRentalStationUris#VehicleRentalStationUris
     BikesAllowed: org.opentripplanner.apis.gtfs.generated.GraphQLTypes.GraphQLBikesAllowed#GraphQLBikesAllowed
-    BookingInfo: org.opentripplanner.model.BookingInfo
-    BookingTime: org.opentripplanner.model.BookingTime
+    BookingInfo: org.opentripplanner.transit.model.timetable.booking.BookingInfo
+    BookingTime: org.opentripplanner.transit.model.timetable.booking.BookingTime
```

Comment:
```suggestion
    BookingInfo: org.opentripplanner.transit.model.timetable.booking.BookingInfo#BookingInfo
    BookingTime: org.opentripplanner.transit.model.timetable.booking.BookingTime#BookingTime
```

Since you are using the shortened type name in the generated file file abive, it's best to also use the shortened name here.

Reply:
This is acualy not ok, NO internal classes should be used as a base for generating API classes - they may change at any point in time. They need to be represented by an API DTO and mapped into that. I see this applay to other classes here as well - I would like to discuss this in a dev meeting.

Reply:
Such a layer exists: https://github.com/opentripplanner/OpenTripPlanner/blob/e68eb35561fea48a99182d7b9aedac8ac7e87f26/src/main/java/org/opentripplanner/apis/gtfs/datafetchers/BookingTimeImpl.java#L9

I'm not sure I love the name `Impl` but it's there.

Reply:
These "Impls" take a core model object and convert it to what the GraphQL API expects.